cram_pr2/cram_pr2_shopping_demo/launch/world.launch

```diff
@@ -4,6 +4,10 @@
     <!-- uploading the shelf to the param server -->
     <param name="shelf_description" command="$(find xacro)/xacro.py
         '$(find cram_pr2_shopping_demo)/resource/shelf.urdf'"/>
+    <!-- uploading basket urdf to param server -->
+    <param name="basket_description" command="$(find xacro)/xacro.py
+        '$(find cram_pr2_shopping_demo)/resource/basket_urdf.urdf'"/>
+
```

Comment:
If we are using primitive shapes now, we don't need a basket urdf, I would get rid of this. If you want to keep it, at least comment it out and write a comment saying that the URDF is not used anymore for X and Y reasons.